src/app/pages/dashboard/dashboard.page.scss

```diff
@@ -81,6 +117,47 @@
     height: 100% !important;
   }
 
+  #page-title {
+    color: var(--mes-secondary-header);
+  }
+
+  #burger-menu {
+    color: var(--mes-secondary-header);
+  }
+
+  hr {
+    background-color: var(--gds-grey-2);
+    height: 2px;
+    margin: 0 14px 0;
+  }
+
+  .version-number {
+    color: var(--mes-secondary-header);
+    margin-top: -4px;
+  }
+
+  .last-sync {
+    color: var(--mes-secondary-header);
+    margin-top: -4px;
+  }
+
+  #unauthenticated-mode-indicator {
+    text-align: center;
+    padding: 16px 0;
+    background: var(--gds-red);
+
+    #unauth-text {
+      color: var(--mes-theme-main-colour);
+      font-size: 20px;
+      font-weight: normal;
+      letter-spacing: 0.34px;
+    }
+  }
+
```

Comment:
Same as above, I didnt think we would be changing any styles, just inverting colours.